components/Tabs.js

```diff
@@ -1,46 +1,82 @@
 import Link from "@components/Link";
+import { Listbox } from "@headlessui/react";
+import { CheckIcon, ChevronUpDownIcon } from "@heroicons/react/20/solid";
```

Comment:
Please specify the full import as this creates a file problem because all are imported 

for example...
```js
import RocketLaunchIcon from "@heroicons/react/20/solid/RocketLaunchIcon";
```